manifests/bind_event.pp

```diff
@@ -0,0 +1,34 @@
+# Define: qpid::bind_event
+#
+# This define binds the event queue with the correct messages
+#
+# === Parameters
+#
+# $ssl_cert::                   SSL cert to use for qpid-config commands
+#
+# $queue::                      Name of the event queue
+#
+# $hostname::                   Set to localhost for qpid-config operations
+#
+# $port::			Port that qpid is listening on
```

Comment:
I'd suggest reviewing diffs yourself before asking for reviews so we have less back-and-forth. Indentation is wrong here.